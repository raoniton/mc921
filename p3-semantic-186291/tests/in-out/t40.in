class Main {
    public static void main(String[] args) {
        int x = 2;
        print(x.value);
    }
}