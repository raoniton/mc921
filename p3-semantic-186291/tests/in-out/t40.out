SemanticError: The type of x must be a class @ 4:15
